src/pkg/dns/resolver_integration_test.go

```diff
@@ -43,4 +43,15 @@ func TestRootResolver(t *testing.T) {
 			t.Errorf("Expected same IP addresses, got %v <> %v", ips, nIPs)
 		}
 	})
+
+	t.Run("LookupIPAddr follows multiple CNAME redirects", func(t *testing.T) {
+		r := RootResolver{}
+		ips, err := r.LookupIPAddr(context.Background(), "cname-a.gnafed.click")
```

Comment:
add to Pulumi

Reply:
Added: https://github.com/DefangLabs/defang-mvp/pull/995